File: js/cw_05.js
// https://www.codewars.com/kata/54ba84be607a92aa900000f1
// Isograms

function isIsogram(str){
  let splittedStr = str.toLowerCase().split('');
  let isIsogram = true;
  for (let char of splittedStr) {
    if (splittedStr.filter((elem) => elem == char).length > 1) {
      isIsogram = false;
    }
  }
  return isIsogram;
}

File: py/cw_05.py
# https://www.codewars.com/kata/520b9d2ad5c005041100000f
# Simple Pig Latin

import re
def pig_it(text):
    text = text.split()
    for index,word in enumerate(text):
        if re.search('[A-Za-z]',word):
            head, *tail = word
            text[index] = "".join(tail) + head + 'ay'
    return " ".join(text)